content/hardware/05.pro-solutions/solutions-and-kits/stella/tutorials/01.user-manual/content.md

```diff
@@ -1,1192 +1,1209 @@
+
+2. **Setup and Initialization**
+
+The setup process for UWB communication differs between the two devices due to their different roles in the ranging session:
+
+**Portenta UWB Shield (Controlee/Responder):**
+
+```arduino
+// Configure the UWB session
+UWBSession session1;
+session1.sessionID(0x11223344);  // Unique identifier for this session
+session1.sessionType(uwb::SessionType::RANGING);
+    
+// Set application parameters
+if(!session1.appParams.addOrUpdateParam(AppConfigId::NO_OF_CONTROLEES,1))
+  Serial.println("could not add to app params");
+if(!session1.appParams.destinationMacAddr(dstAddr))
+  Serial.println("could not add to app params");
+    
+// Apply default values for measurement repetition rate and antenna config
+session1.applyDefaults();
+// Configure ranging parameters
+session1.rangingParams.deviceMacAddr(srcAddr);
+session1.rangingParams.deviceRole(uwb::DeviceRole::RESPONDER);
+session1.rangingParams.deviceType(uwb::DeviceType::Controlee);
+session1.rangingParams.multiNodeMode(uwb::MultiNodeMode::UNICAST);
+session1.rangingParams.rangingRoundUsage(uwb::RangingMethod::DS_TWR);
+session1.rangingParams.scheduledMode(uwb::ScheduledMode::TIME_SCHEDULED);
+  
+// Add the session to the manager and start it
+UWBSessionManager.addSession(session1);
+session1.init();
+session1.start();
+```
+
+**Arduino Stella (Controller/Initiator):**
+
+```arduino
+// Configure the UWB session
+UWBSession session1;
+session1.sessionID(0x11223344);
+session1.sessionType(UWBD_RANGING_SESSION);
+    
+// Set application parameters
+if(!session1.appParams.addOrUpdateParam(UWB_SET_APP_PARAM_VALUE(NO_OF_CONTROLEES,1)))
+  Serial.println("could not add to app params");
+if(!session1.appParams.destinationMacAddr(dstAddr))
+  Serial.println("could not add to app params");
+    
+// Apply default values for measurement repetition rate and antenna config
+session1.applyDefaults();
+// Configure ranging parameters
+session1.rangingParams.deviceMacAddr(srcAddr);
+session1.rangingParams.deviceRole(kUWB_DeviceRole_Initiator);
+session1.rangingParams.deviceType(kUWB_DeviceType_Controller);
+session1.rangingParams.multiNodeMode(kUWB_MultiNodeMode_UniCast);
+session1.rangingParams.rangingRoundUsage(kUWB_RangingRoundUsage_DS_TWR);
+session1.rangingParams.scheduledMode(kUWB_ScheduledMode_TimeScheduled);
+  
+// Add the session to the manager and start it
+UWBSessionManager.addSession(session1);
+session1.init();
+session1.start();
+```
+
+***<strong>Important note:</strong> The session configuration is more detailed for the Portenta UWB Shield because it explicitly defines all the ranging parameters. The Arduino Stella uses the simplified `UWBTracker` class, which automatically sets up the device as a Controller/Initiator with appropriate defaults. However, both devices must use the same session ID (`0x11223344` in this example) to communicate with each other. This session ID is a shared identifier for the ranging session between these devices.***

```

Comment:
```suggestion
***<strong>Important note:</strong> The session configuration is more detailed for the Portenta UWB Shield because it explicitly defines the ranging parameters. The Arduino Stella uses the `UWBSession` class and relies on the library’s default settings initially to operate as the Controller/Initiator. However, both devices must use the same session ID (`0x11223344` in this example) to communicate with each other. This session ID is a shared identifier for the ranging session between these devices.***

```
A suggestion, since we use UWBSession and UWBTracker is not used anymore if I'm not wrong 🤔